### PR TITLE
feat(AASTU-web-G2): Create Footer

### PR DESCRIPTION
**Pull request by wubeZ**
Created the footer component

### CODE REVIEW

**Review by natanim-ashenafi — APPROVED** [review]
The code is well written.

File: aastu/web/group-2/app/components/footer.tsx [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+            </nav>
+        </div>
+      </div>
+
+      <div className="flex flex-col-reverse items-center lg:flex-row justify-between px-[40px] py-[35px] border-t-[#C7C7C7] border-t-[0.4px]">
+        <div className="mt-8 md:font-secondaryFont font-light text-base">
+          <span className="text-xl text-[#555555]">&copy;</span> 2020 Africa to Silicon Valley ,
+          Inc. All right reserved.
+        </div>
+        <div className="flex h-[20px] w-[280px] list-none items-center justify-between">
```
Comment (abel-getahun-a2sv) [inline aastu/web/group-2/app/components/footer.tsx:54]: It looks well done to me and make sure to add the navigation URL for each of footer links. Nice work wube!

**Review by natanim-ashenafi — APPROVED** [review]
The code follows best practices.

**Review by abel-getahun-a2sv — APPROVED** [review]
Nicely done work, keep it up.

**Review by natanim-ashenafi — COMMENTED** [review]
The code looks good to me.

**Pull request closed without merging**